docs/service_accounts.md

```diff
@@ -0,0 +1,112 @@
+# Tiller and Service Accounts
+
+In Kubernetes, granting a role to an application-specific service account is a best practice to ensure that your application is operating in the scope that you have specified. Read more about service account permissions in Kubernetes [in the official docs](https://kubernetes.io/docs/admin/authorization/rbac/#service-account-permissions). Bitnami also has a fantastic guide for [configuring RBAC in your cluster](https://docs.bitnami.com/kubernetes/how-to/configure-rbac-in-your-kubernetes-cluster/) that takes you through RBAC basics.
```

Comment:
Nit: Change "Read more about service account permissions in Kubernetes [in the official docs]" to "Read more about service account permissions [in the official Kubernetes docs]". Reads a little cleaner.